web-rdfs/include/http_handlers.h

```diff
@@ -15,6 +15,16 @@ using HttpsServer = SimpleWeb::Server<SimpleWeb::HTTPS>;
 
 void setZk(zkclient::ZkNnClient *zk_arg);
 
+/**
+ * Handler invoked when the client issues a GET request to the web server root
```

Comment:
Does this work for PUT, POST, and DELETE too?

Reply:
No. The browser has no way of issuing these verbs when accessing the client webpage anyway.